modules/git/repo_commit.go

```diff
@@ -142,6 +142,9 @@ func (repo *Repository) searchCommits(id ObjectID, opts SearchCommitsOptions) ([
 		cmd.AddArguments("--all")
 	}
 
+	// interpret search string keywords as string instead of regex
+	cmd.AddArguments("-F")
```

Comment:
Maybe it could use the long argument, it reads easier.

```suggestion
	cmd.AddArguments("--fixed-strings")
```

ps: tests don't seem very necessary this time, good to have, while it also LGTM without test since it is quite simple and won't cause serious regression.